lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_FallingSnowLevel.py

```diff
@@ -445,60 +444,56 @@ class Test_process(IrisTest):
     def setUp(self):
         """Set up cubes."""
```

Comment:
Again it wouldn't hurt to put a bit more of a docstring here

Reply:
Extended slightly to describe what is being set up.